apps/app/components/issues/EditorBubbleMenu.tsx

```diff
@@ -0,0 +1,116 @@
+import { BubbleMenu, BubbleMenuProps } from "@tiptap/react";
```

Comment:
Rename this component file name as the rest of the components with small letters.